HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,36 @@
+def extraArgs(neighbors,distance):
+	"""If a node is in neighbors, and it hasn't been visited, return it. Else return"""
+	def updateNode(n):
```

Comment:
I was having a bug where the the current value of i was not being passed in when calling:
graph = graph.map(updateNode(neighbors,i))
 Instead, it would pass in the value of i only when I performed an action such as filter. (I think this has to do with spark's lazy evaluation)
So extraArgs allows me to fix this